code/game/objects/items/rogueitems/keyrings.dm

```diff
@@ -125,19 +125,19 @@
 	keys = list(/obj/item/key/sheriff_office, /obj/item/key/town_armory, /obj/item/key/town_barracks, /obj/item/key/town_dungeon)
 
 /obj/item/storage/keyring/man_at_arms
-	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/manor, /obj/item/key/walls)
+	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/manor, /obj/item/key/walls, /obj/item/key/keep_armory)
```

Comment:
Giving the MAA armoury keys is a bit questionable. Probably should be kept to Captain/knights no?